dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -0,0 +1,365 @@
+            }
+          },
+          "title": "CPU under-provisioned"
+        },
+        "width": 2,
+        "xPos": 6,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "thresholds": [
+              {
+                "color": "RED",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0.1
+              }
+            ],
+            "timeSeriesQuery": {
+              "timeSeriesQueryLanguage": "{ all_non_besteffort_workloads:\n    fetch k8s_container\n    | metric 'kubernetes.io/container/memory/request_bytes'\n    | filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system'\n    | group_by 5m, [value_request_bytes_mean: mean(value.request_bytes)]\n    | every 5m\n    | group_by\n        [resource.project_id, resource.location, resource.cluster_name,\n         resource.namespace_name,\n         metadata.system.top_level_controller_name:\n           metadata.system_labels.top_level_controller_name],\n        [value_request_bytes_mean_mean: mean(value_request_bytes_mean)]\n    | filter val() != 0\n    | group_by [], [row_count: row_count()]\n; burstable_workload_risk:\n    fetch k8s_container\n    | metric 'kubernetes.io/container/memory/request_utilization'\n    | filter\n        (resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system'\n         && metric.memory_type == 'non-evictable')\n    | group_by 3d,\n        [value_request_utilization_mean: mean(value.request_utilization)]\n    | every 5m\n    | group_by\n        [resource.project_id, resource.location, resource.cluster_name,\n         resource.namespace_name,\n         metadata.system_labels.top_level_controller_name,\n         metadata.system_labels.top_level_controller_type],\n        [value_request_utilization_mean: mean(value_request_utilization_mean)]\n    | filter val() >= 1\n    | group_by [], [row_count: row_count()] }\n| join\n| value\n    [v_0:\n       scale(\n         div(burstable_workload_risk.row_count,\n           all_non_besteffort_workloads.row_count),\n         '%')]"
+            }
+          },
+          "title": "MEM under-provisioned"
```

Comment:
Let's be consistent and use Memory everywhere instead of MEM

Reply:
updated